.github/workflows/build_wheels_windows.yml

```
@@ -0,0 +1,166 @@
+              if ($inPackage -and !$patched -and $line -match '^version = ".*"') {
+                  "version = `"$cargoVersion`""
+                  $patched = $true
+              } else { $line }
+          }
+          Set-Content "$cargoPath" $newLines
+
+      - name: Setup Rust Toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+
+      - name: Install dependencies
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC1001:info:3:52: This \v will be a regular 'v' in this context

(shellcheck)